code/game/verbs/ooc.dm

```diff
@@ -138,3 +138,23 @@ var/global/normal_ooc_colour = "#002eb8"
 		return
 
 	show_note(usr, null, 1)
+
+/client/proc/ignore_key(client)
+	var/client/C = client
+	prefs.ignoring ^= C.key
+	src << "You are [prefs.ignoring.Find(src.key) ? "now" : "no longer"] ignoring [C.key] on the OOC channel."
+	prefs.save_preferences()
+
+/client/verb/select_ignore()
+	set name = "Ignore"
+	set category = "OOC"
+	set desc ="Ignore a player's messages on the OOC channel"
+
+	var/list/keys = list()

```

Comment:
It's from the ref and the bluebook, and the redbook doesn't mention it as erroneous.
Not to mention every other var you use defaults to in world, I very much doubt client would have it's own snowflake code in the base for list proc.